### PR TITLE
changes related to $schema refererence update related to SPFx 1.18.x

### PR DESCRIPTION
**Pull request by HarminderSethi**
## Category

- [x] Content fix
- [ ] New article

## Related issues

- N/A

### CODE REVIEW

**Comment by HarminderSethi** [comment]
for the warning in the build status [docs/spfx/viva/get-started/actions/geolocation/GeolocationTutorial.md], I was not sure if it is requied to change the alt text 'Card appearance after introducing changes in the quick-view' for one image

**Comment by andrewconnell** [comment]
@HarminderSethi  Yes, that must be fixed. For those who are vision impaired and use screenreaders, it's bad form to use the exact same description on two different images.

File: docs/spfx/viva/get-started/actions/geolocation/GeolocationTutorial.md
Comment (andrewconnell) [inline docs/spfx/viva/get-started/actions/geolocation/GeolocationTutorial.md:1]: Please correct the automatic finding that two images use the same ALT text

**Review by andrewconnell — CHANGES_REQUESTED** [review]

File: docs/spfx/viva/get-started/actions/geolocation/GeolocationTutorial.md
Comment (HarminderSethi) [reply docs/spfx/viva/get-started/actions/geolocation/GeolocationTutorial.md:1]: @andrewconnell I have made the required changes. please review once.

**Review by andrewconnell — APPROVED** [review]

**Pull request merged**